index.html

```diff
@@ -0,0 +1,392 @@
+
+                        <div class="card__footer">
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (12).svg" alt=""></a>
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (13).svg" alt=""></a>
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (14).svg" alt=""></a>
+
+                        </div>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <img class="card__img" src="/img/cover (2).png" alt="">
+                    <div class="card__body">
+                        <h5 class="text-card">Avie Beaton</h5>
+                        <p class="professional professional--color">CO Founder</p>
+
+                        <div class="card__footer">
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (12).svg" alt=""></a>
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (13).svg" alt=""></a>
+                            <a class="card__icon" href="#"><img src="/vectors/Vector (14).svg" alt=""></a>
+                        </div>
+                    </div>
+                </div>
+        </section>
+
+        <section class="main-page__form-button">
+
+            <div class="main-page__form">
+
+                <div class="main-page__contact">
+                    <p class="main-page__contact-text">Contact Us</p>
+                    <h2 class="main-page__contact-header">Make an Appointment</h2>
+                </div>
+
+                <form class="form">
+
+                    <div class="first-input">
+                        <div>
+                            <input type="text" id="name" class="form-input" placeholder="Full Name">
+                        </div>
+                        <div>
+                            <input type="text" id="email" class="form-input" placeholder="example@gmail.com">
+                        </div>
+                    </div>
+
+                    <div class="second-input">
+                        <div>
+                            <input type="text" id="name" class="form-input" placeholder="Please Select">
+
+                        </div>
+                        <div>
+                            <input type="text" id="email" class="form-input" placeholder="4:00 Available">
+                        </div>
+                    </div>
+
+                    <div class="form-group">
+                        <textarea class="form__text" id="exampleFormControlTextarea1"
+                            placeholder="Message..."></textarea>
+                    </div>
+
+                    <div>
+                        <button class="btn btn--rectangle">Book Appointment</button>
+                    </div>
+                </form>
+            </div>
+
+            <div class="main-page-background main-page-background__color">
```

Comment:
`main-page-background main-page-background__color` is block and element on the same DOM Element